settings.gradle

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/8.0.2/userguide/multi_project_builds.html
+ */
+import java.nio.file.Paths
+
+plugins {
+    id("com.gradle.enterprise") version("3.16.2")
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.7.0'
+}
+
+void addSubProjects(String path, File dir) {
+    if (dir.isDirectory() == false) {
+        return;
+    }
+    if (dir.name == 'buildSrc') {
+        return;
+    }
+    if (new File(dir, 'build.gradle').exists() == false) {
+        return;
+    }
+    if (findProject(dir) != null) {
+        return;
+    }
+
+    final String projectName = "${path}:${dir.name}"
+    include projectName
+
+    project(projectName).projectDir = dir
+
+    for (File subdir : dir.listFiles()) {
+        addSubProjects(projectName, subdir)
+    }
+}
+
+rootProject.name = 'MigrationsAssistant'
+include 'TrafficCapture:captureKafkaOffloader'
+include 'TrafficCapture:captureOffloader'
+include 'TrafficCapture:captureProtobufs'
+include 'TrafficCapture:nettyWireLogging'
+include 'TrafficCapture:trafficCaptureDependencies'
```

Comment:
They should and soon, but I wanted to leave the people working on RFS currently to make that call.  
I'm hoping that this commit/merge can open the door to changes like that but also not require them to be made immediately.

Reply:
What do you think about moving the dependencies project to the top level now to minimize future changes on the projects under TrafficCapture.  Then how/when RFS moves it does on its own timeline?

Reply:
I'll move it up, but I'll need a new name other than trafficCaptureDependencies.  Does "commonDependencyVersionConstraints" work?  commonDependencies sounds like the project should be logging, coreUtilities, etc.  But I don't like the idea that we need any dependencies at all in any of our projects, hence the reason that I've added the extra words.

QQ: If these tools were in separate repos, would you still want to see common dependencies locked across repos/projects?  Would you have a replayer project, capture project, & RFS project all take a dependency on a common dependency project?  I think that it would still make sense, but it does (literally & intentionally) bind constraints on separate projects.

Reply:
What do you think about a shorter `dependencyConstraints`? IMO when/if we get to another repo that might be a good time to reconsider the naming convention. 

I'm not too concerned when by looking at the project gradle its clear there is no actual source code and its consistently a single thing - a big list of dependency versions